### ISSUE TITLE
Transparent logo missing in Safari Desktop and Chrome iOS

### ISSUE THREAD

**Issue by baohouse** [brainstormforce/astra#1539]
My website is https://productfy.io

On Chrome Desktop, the logo shows up fine (and you can view the page source and see the IMG tag.

<img width="1590" alt="Screen Shot 2019-12-05 at 2 30 54 PM" src="https://user-images.githubusercontent.com/4228012/70279720-e0e0f480-176b-11ea-9644-9ccd6dc71b88.png">

~~But if I view the page source in Safari, it's not there:~~

<img width="1359" alt="Screen Shot 2019-12-05 at 2 32 10 PM" src="https://user-images.githubusercontent.com/4228012/70279811-0bcb4880-176c-11ea-8ee5-0c5b26ad4d7f.png">

The logo is not present in mobile Chrome, but is present in mobile Safari! I have no idea what's going on. I'm testing with a MacBook Pro on Catalina, and iPhone 11. In the meanwhile, I've had to manually inject the logo via CSS, but the IMG tag is missing if you view the page source in Desktop Safari.

UPDATE: It now shows up in Desktop Safari! Strange. But still not in Chrome on iPhone.

**Comment by baohouse** [comment]
I think I figured it out. So testing on Macbook Pro and iPhone 11, they are using Retina resolutions. So I enabled the "Different Logo for Retina Devices?" option in the Transparent Header configuration, and it showed up correctly. I suspect if you uncheck this box, then the logo won't appear. I would have expected to fallback and use the logo specified above it under "Different Logo for Transparent Header?" if you don't specify a retina-specific logo.
